mars/tests/test_utils.py

```diff
@@ -586,3 +586,8 @@ def __call__(self, *args, **kwargs):
 
     func = Func()
     assert get_func_token_values(func) == [func]
+
+
+@require_ray
+def test_report_event(ray_start_regular):
+    utils.report_event("WARNING", "test_label", "test_message")
```

Comment:
No validations for this test, please change.

Reply:
Fixed it by mock report_event